src/main/java/it/gov/pagopa/ranking/dto/event/EvaluationRankingDTO.java

```diff
@@ -36,4 +36,5 @@ public class EvaluationRankingDTO {
     private List<OnboardingRejectionReason> onboardingRejectionReasons;
     private BigDecimal beneficiaryBudget;
     private String initiativeRewardType;
+    private String organizationName;
```

Comment:
let's keep fields ordered in order to find them more easily, put this after organizationId field